src/Assignment-1/assignment1.html

```diff
@@ -14,33 +14,8 @@
 	<!-- This is just a comment. Ignore this line.-->
 	<!-- Start your code from here -->
 	
-	<!-- this is changing -->
+	
 	<!-- End your code -->
-	<div class="row">
-		<div class="column">
```

Comment:
Please keep class name for all column as col..so that it will help for responsiveness of your webpage